pyface/ui/qt/util/gui_test_assistant.py

```diff
@@ -44,16 +44,22 @@ def setUp(self):
         except ImportError:
             self.traitsui_raise_patch = None
         else:
-            self.traitsui_raise_patch = mock.patch(
-                "traitsui.qt4.ui_base._StickyDialog.raise_"
-            )
+            try:
```

Comment:
It feels as though we should just duplicate the TraitsUI helper in this repository to avoid the `traitsui` imports. But not in this PR.

Reply:
Opened #1225